From 0d258ecd8212baceaf46f7d1fc653cd6c4622c6f Mon Sep 17 00:00:00 2001
From: KoysX <2685697179b@gmail.com>
Date: Tue, 4 Jul 2023 23:02:35 +0800
Subject: [PATCH] Revert-Biometrics-Allow-disabling-of-fingerprint-cle

---
 .../sensors/fingerprint/hidl/Fingerprint21.java           | 8 --------
 1 file changed, 8 deletions(-)

diff --git a/services/core/java/com/android/server/biometrics/sensors/fingerprint/hidl/Fingerprint21.java b/services/core/java/com/android/server/biometrics/sensors/fingerprint/hidl/Fingerprint21.java
index fab3d902..f93281e0 100644
--- a/services/core/java/com/android/server/biometrics/sensors/fingerprint/hidl/Fingerprint21.java
+++ b/services/core/java/com/android/server/biometrics/sensors/fingerprint/hidl/Fingerprint21.java
@@ -128,8 +128,6 @@ public class Fingerprint21 implements IHwBinder.DeathRecipient, ServiceProvider
     private final int mSensorId;
     private final boolean mIsPowerbuttonFps;
 
-    private boolean mCleanup;
-
     private final class BiometricTaskStackListener extends TaskStackListener {
         @Override
         public void onTaskStackChanged() {
@@ -351,9 +349,6 @@ public class Fingerprint21 implements IHwBinder.DeathRecipient, ServiceProvider
             mCurrentUserId = UserHandle.USER_NULL;
         });
 
-        mCleanup = context.getResources().getBoolean(
-                org.lineageos.platform.internal.R.bool.config_cleanupUnusedFingerprints);
-
         try {
             ActivityManager.getService().registerUserSwitchObserver(mUserSwitchObserver, TAG);
         } catch (RemoteException e) {
@@ -739,9 +734,6 @@ public class Fingerprint21 implements IHwBinder.DeathRecipient, ServiceProvider
 
     private void scheduleInternalCleanup(int userId,
             @Nullable ClientMonitorCallback callback) {
-        if (!mCleanup) {
-            return;
-        }
         mHandler.post(() -> {
             scheduleUpdateActiveUserWithoutHandler(userId);
 
-- 
2.20.1

